docs/source/docker_only.rst

```diff
@@ -0,0 +1,59 @@
+It is possible to develop with dallinger using only docker as a prerequisite.
+This guide goes through the necessary steps to achieve this.
+
+Set up services
+===============
+
+Start services needed by dallinger:
+
+.. code-block:: shell
+
+    docker run -d --name dallinger_redis --network=dallinger -v dallinger_redis:/data redis redis-server --appendonly yes
+    docker run -d --name dallinger_postgres --network=dallinger -e POSTGRES_USER=dallinger -e POSTGRES_PASSWORD=dallinger -e POSTGRES_DB=dallinger -v dallinger_postgres:/var/lib/postgresql/data postgres:12
+
+Start adminer:
+
+.. code-block:: shell
+
+    docker run -d --name dallinger_adminer --network dallinger --link dallinger_postgres:db adminer
+
+Find the adminer ip:
+
+.. code-block:: shell
+
+    docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' dallinger_adminer
+
+Connect to adminer by visiting http://<result-of-docker-inspect>:8080
```

Comment:
Not just http://localhost:8000/ ?

Reply:
See [this other comment](https://github.com/Dallinger/Dallinger/pull/4293#discussion_r955936799).